src/TabCompletion/Matcher/AbstractDefaultParametersMatcher.php

```diff
@@ -29,7 +29,7 @@ public function getDefaultParameterCompletion(array $reflectionParameters): arra
 
             $defaultValue = $this->valueToShortString($parameter->getDefaultValue());
 
-            $parametersProcessed[] = "\${$parameter->getName()} = $defaultValue";
+            $parametersProcessed[] = "\{${$parameter->getName()}} = $defaultValue";
```

Comment:
You are right, it indeed was a mistake because I didn't realize the string was meant to have the dollar sign there. I reverted that change for now, but also completely agree with the `sprintf` idea.

Perhaps, we can change it to something like this?

```php
$parametersProcessed[] = sprintf('$%s = %s', $parameter->getName(), $defaultValue);
```